web/shapeworks/public/forms/groom.json

```diff
@@ -241,7 +274,8 @@
                     "default": "shape_1",
                     "x-cols": 6,
                     "x-display": "custom-conditional",
-                    "x-display-if":"(model) => model.section_2.reflect"
+                    "x-display-if":"(model) => model.section_2.reflect",
```

Comment:
Should these actually be section 3 (also in the alignment controls)?

Reply:
Do you mean the fields for `reflect`, `reflect_column`, and `reflect_axis`? I was mocking the desktop app, but if the shapeworks folks agree it works better in section 3, we can easily move them.

Reply:
When I check the reflect box, I don't see `reflect_column` or `reflect_axis` appear.  If these use `model.section_3`, then they do appear.  In the desktop app, they appear in the Alignment section, which is section 3 here.

Reply:
Ah I see what you mean. Fixed in [f7ebc05](https://github.com/girder/shapeworks-cloud/pull/176/commits/f7ebc052df07492aa4235aa29499042918e2680b)